tests/unit-tests/formatting/functions.php

```diff
@@ -685,7 +685,11 @@ public function test_wc_timezone_string() {
 
 		// Test with manually set UTC offset.
 		update_option( 'gmt_offset', -4 );
-		$this->assertNotEquals( 'UTC', wc_timezone_string() );
+		$this->assertEquals( 'UTC', wc_timezone_string() );
```

Comment:
I don't think this test is correct. Shouldn't it be checking if `wc_timezone_string()` return value equals to `America/Anguilla` instead of `UTC`? At least that is what this function returns on `master`.

If we make the change that I mentioned above, the test will fail which as far as I can see means that we still need to make a few more modifications to `wc_timezone_string()` to make it work as before and at the same time support float values for timezones.